src/orchard/info.clj

```diff
@@ -98,37 +98,47 @@
      ;; a special symbol - always use :unqualified-sym
      (some-> (cljs-ana/special-meta env unqualified-sym)
              (cljs-meta/normalize-var-meta))
-     ;; an NS
+
+     ;; a NS
```

Comment:
Well ackhually, `an NS` is correct.

Reply:
Indeed, it's pronounced as `en es`, so the normal rules about consonants don't apply here.